dev/docker/doris/start.sh

```diff
@@ -37,6 +36,7 @@ PRIORITY_NETWORKS=$(echo "${CONTAINER_IP}" | awk -F '.' '{print$1"."$2"."$3".0/2
 echo "add priority_networks = ${PRIORITY_NETWORKS} to fe.conf & be.conf"
 echo "priority_networks = ${PRIORITY_NETWORKS}" >> ${DORIS_FE_HOME}/conf/fe.conf
 echo "priority_networks = ${PRIORITY_NETWORKS}" >> ${DORIS_BE_HOME}/conf/be.conf
+echo "disable_storage_medium_check = true" >> ${DORIS_FE_HOME}/conf/fe.conf
```

Comment:
Will this configuration prevent the check for left storage when starting?

Reply:
Yes, It's no need to check this. 

> If disable_storage_medium_check is true, ReportHandler would not check tablet's storage medium and disable storage cool down function, the default value is false. You can set the value true when you don't care what the storage medium of the tablet is.

When we have SSD and HDD on same machine, this check is helpful.

When we use in CI, this config may cause BE refuse request, and finally make `create table` failed probabilistic (when the container has just been started)

I am checking it in CI